actions/common/search_models.go

```diff
@@ -20,3 +21,30 @@ type SearchModel[TFilter any] struct {
 	// Pagination and sorting options to streamline data exploration and analysis
 	QueryParams *datastore.QueryParams `json:"params,omitempty" swaggertype:"object,string" example:"page:1,page_size:10,order_by_field:created_at,order_by_direction:desc"`
 }
+
+// PagedResponse object to use when returning database records in paged format
+type PagedResponse[content any] struct {
+	// List of records for the response
+	Content []content `json:"content"`
+	// Pagination details
+	Page Page `json:"page"`
+}
+
+// Page object to use when limiting and sorting database query results
+type Page struct {
+	// Field by which to order the results
+	OrderByField *string `json:"orderByField"`
+	// Direction in which to order the results ASC/DSC
+	SortDirection *string `json:"sortDirection"`
+	// Total count of elements
+	TotalElements int64 `json:"totalElements"`
+	// Total number of possible pages
+	TotalPages int `json:"totalPages"`
+	// Size of the page
+	Size int `json:"size"`
+	// Page number
+	Number int `json:"number"`
+}
```

Comment:
I'm just wondering if it would be beneficial to move such models to the `models` package. Thanks to that we could use them in `go-client`.

Reply:
Done